lib/vmq_elixir_plugin.ex

```diff
@@ -22,17 +20,7 @@ defmodule VmqElixirPlugin do
   Application start function.
   """
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
-
-    children = [
-      # Define workers and child supervisors to be supervised
-      # worker(VmqElixirPlugin.Worker, [arg1, arg2, arg3])
-    ]
-
-    # See http://elixir-lang.org/docs/stable/elixir/Supervisor.html
-    # for other strategies and supported options
-    opts = [strategy: :one_for_one, name: VmqElixirPlugin.Supervisor]
-    Supervisor.start_link(children, opts)
+    :ok
```

Comment:
:ok is not allowed to be returned here. I think we should convert the plugin to a library application by removing `mod: {VmqElixirPlugin, []},` from mix.exs and then we can even remove the `start/2`function here as well as it will never be called when the application is started.